src/Util/ClassSourceManipulator.php

```diff
@@ -197,8 +233,36 @@ public function addSetter(string $propertyName, $type, bool $isNullable, array $
         $this->addMethod($builder->getNode());
     }
 
-    public function addMethodBuilder(Builder\Method $methodBuilder)
+    public function addConstructor(array $params = [], string $methodBody = null)
     {
+        if ($this->getConstructorNode()) {
+            throw new \LogicException('Constructor already exists.', 1);
+        }
+
+        $methodBuilder = $this->createMethodBuilder('__construct', null, false, []);
```

Comment:
Last argument is optional, so it's not needed here.

Reply:
Okay